run_dir/design/deliveries.html

```diff
@@ -197,83 +196,101 @@ <h3>
+                           <button class="btn btn-sm btn-imp mr-2" value="Important" data-toggle="tooltip" title="For when a note needs to be highlighted">Important <span class="fa fa-exclamation-circle"></span></button>
+                           <button class="btn btn-sm btn-devi" value="Deviation" data-toggle="tooltip" title="For notes about a deviation">Deviation <span class="fa fa-frown"</span></button>
+                        </div>
+                      </div>
+                  </div>
+                    <div class="row mt-3">
                         <div class="col-md-6">
                             <h4>Write Running Note</h4>
                             <textarea rows="5" class="form-control" id="new_note_text" style="height:97px;"></textarea>
-
-                            <label>Category :</label>
-                            <select name="rn_cat" class="form-control" id="rn_category">
-                                <option value="Workset">Workset</option>
-                                <option value="Flowcell">Flowcell</option>
-                                <option value="Meeting">Meeting</option>
-                                <option value="Decision">Decision</option>
-                                <option value="Bioinformatics">Bioinformatics</option>
-                                <option value="User Communication">User Communication</option>
-                                <option value="" selected="selected">None</option>
-                            </select>
                         </div>
                         <div class="col-md-6">
                             <h4>Preview</h4>
-                            <div class="panel panel-default" id="running_note_preview_panel">
-                                <div class="panel-heading"><a href="#"></a> - <span class="todays_date">Date</span></div>
-                                <div class="panel-body" id="running_note_preview_body">
+                            <div class="card" id="running_note_preview_panel">
+                                <div class="card-header">
+                                  <a class="text-decoration-none" href="#">{{ user.name }}</a> - <span class="todays_date">Date</span>
+                                  <span id="preview_category"> - Category</span>
+                                </div>
+                                <div class="card-body" id="running_note_preview_body">
                                     <p class="text-muted"><em>Nothing to preview..</em></p>
                                 </div>
                             </div>
                         </div>
                     </div>
-                    <div class="text-right" style="margin-top:15px;">
+                    <div class="row mt-3">
+                      <div class="col-md-6 text-right" style="margin-top:15px;">
```

Comment:
I guess the `style="margin-top:15px` can be accomplished with one of the margin classes now?

Reply:
Done!